### DISCUSSION TITLE
Container unexpectedly stopping catching `SIGWINCH`

### DISCUSSION THREAD

**Discussion by JeffResc** [docker-library/wordpress#698] [General]
Hi all,

My container usually works fine, but recently I've been coming across the following error in my logs after realizing the the Wordpress container has gone offline:
```
[Mon Mar 21 18:33:52.216317 2022] [mpm_prefork:notice] [pid 1] AH00170: caught SIGWINCH, shutting down gracefully
```

The container is also set to restart unless it is stopped, which it was not stopped by any users intentionally. I tried to do some digging, but was unable to find the root cause of the error, or how to prevent it from happening in the future. Any help on this would be greatly appreciated.

**Comment by yosifkit** [comment]
The `apache` graceful shutdown signal is `SIGWINCH`. It is the image-defined `STOPSIGNAL`. So it could be getting this signal from a load balancer considering it unhealthy (like https://github.com/docker-library/php/issues/892), or if you are running the container interactively, it would get the signal from resizing the window.

**Comment by JeffResc** [comment]
> The `apache` graceful shutdown signal is `SIGWINCH`. It is the image-defined `STOPSIGNAL`. So it could be getting this signal from a load balancer considering it unhealthy (like [docker-library/php#892](https://github.com/docker-library/php/issues/892)), or if you are running the container interactively, it would get the signal from resizing the window.

Thanks for elaborating on what could be the cause. While this container does sit behind the Traefik reverse proxy, I am not using it as a load balancer or have any other Wordpress containers running for it to balance to. I don’t believe I have any settings that monitor the container for health to stop/restart it or anything like that.

Additionally, I have it running from docker-compose in the background and rarely enter into the container outside of maintenance, so I don’t believe a terminal resize would have caused this either as no one was logged into the server at the time of the shutdown.

I’ll double check my Traefik configuration to ensure it didn’t do it, but I don’t believe it is setup to start/stop/restart any containers. I’ll let you know if I find anything in terms of that.

**Reply by carlosjfcasero** [reply]
Hi, I'm facing the same issue. Did you manage to solve it?

Thanks in advance!

**Reply by JeffResc** [reply]
Hi @carlosjfcasero —

Unfortunately, I wasn't able to determine the issue on this one, I simply configured the container to auto restart on failure. I'm still not sure exactly what's causing the issue, sorry.

Best of luck, and if you find anything, please let me know!
